build.gradle.kts

```diff
@@ -17,13 +17,20 @@ java {
 
 repositories {
     mavenCentral()
+    gradlePluginPortal()
+    maven("https://repo.spring.io/milestone")
 }
 
 dependencies {
     implementation("org.springframework.boot:spring-boot-starter-web")
     implementation("org.jetbrains.kotlin:kotlin-reflect")
     implementation("com.squareup.okhttp3:okhttp:4.12.0")
     implementation("org.mariadb.jdbc:mariadb-java-client:3.5.1")
+    implementation("com.squareup.okhttp3:okhttp:4.12.0")
+    implementation("org.apache.poi:poi:5.3.0")
+    implementation("org.apache.poi:poi-ooxml:5.3.0")
+    implementation("com.opencsv:opencsv:5.9")
     compileOnly("org.projectlombok:lombok")
     annotationProcessor("org.projectlombok:lombok")
+    testImplementation("org.springframework.boot:spring-boot-starter-test")
```

Comment:
I think we are not using this test library at all and it can be removed.

Reply:
Fixed.